### PR TITLE
[DOCS] Adjust Templates for PRs and Issues

### PR DESCRIPTION
**Pull request by linawolf**
- Supported TYPO3 versions are 11 and 12
- Supported TYPO3 versions PHP 7.4 - 8.2

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 1]
```diff
@@ -11,8 +11,8 @@ assignees: ''
 
 ## Prerequisites
 
-* [ ] Can you reproduce the problem on TYPO3 v10.4 LTS
-* [ ] Can you reproduce the problem on TYPO3 v11.3
+* [ ] Can you reproduce the problem on TYPO3 v11.5 LTS
+* [ ] Can you reproduce the problem on TYPO3 v12.4
```
Comment (gilbertsoft) [inline .github/ISSUE_TEMPLATE/bug-report.md:15]:
```suggestion
* [ ] Can you reproduce the problem on TYPO3 v12.4 LTS
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -8,13 +8,14 @@
...
 * [ ] Changes have been tested on TYPO3 v11.5 LTS
-* [ ] Changes have been tested on PHP 7.2
-* [ ] Changes have been tested on PHP 7.3
+* [ ] Changes have been tested on TYPO3 v12.4 LTS
 * [ ] Changes have been tested on PHP 7.4
 * [ ] Changes have been tested on PHP 8.0
+* [ ] Changes have been tested on PHP 8.1
+* [ ] Changes have been tested on PHP 8.2
 * [ ] Changes have been checked for CGL compliance `php-cs-fixer fix`
+* [ ] CSS has been rebuild (only if there are SCSS changes `cd Build; npm ci; ./node_modules/.bin/grunt build`)
```
Comment (gilbertsoft) [inline .github/PULL_REQUEST_TEMPLATE.md:18]:
```suggestion
* [ ] CSS has been rebuilt (only if there are SCSS changes `cd Build; npm ci; npm build`)
```

**Review by gilbertsoft — CHANGES_REQUESTED** [review]

**Comment by linawolf** [comment]
@gilbertsoft `npm build` does not seem to work:

lina@bootstrap-package-web:/var/www/html/Build$ npm build
Unknown command: "build"

Did you mean this?
    npm run build # run the "build" package script

**Comment by gilbertsoft** [comment]
> Did you mean this? npm run build # run the "build" package script

Exactly, forgot the run

**Review by gilbertsoft — APPROVED** [review]

**Pull request merged**